<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifiereq</class>
 <widget class="QDialog" name="modifiereq">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/images/backg2.png);</string>
  </property>
  <widget class="QLineEdit" name="disponibilite2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-image: url(:/images/images/WHITE.png);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifier1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
color: #ffffff;
border: 0px solid ;
font: 10pt &quot;Segoe UI Black&quot;;
border-radius: 26px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.65, stop: 0 #1E73E8, stop: 1 #1E73E8);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.65, stop: 0 #5E9DF5, stop: 1 #5E9DF5);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.65, stop: 0 #0936F5, stop: 1 #0936F5);
}</string>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:rgb(255, 255, 255);
color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>id</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="model2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-image: url(:/images/images/WHITE.png);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="marque2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-image: url(:/images/images/WHITE.png);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_equipement2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-image: url(:/images/images/WHITE.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>400</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:rgb(255, 255, 255);
color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>disponibilite</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:rgb(255, 255, 255);
color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>model</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:rgb(255, 255, 255);
color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>prix</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="etat2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-image: url(:/images/images/WHITE.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:rgb(255, 255, 255);
color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>etat</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="prix2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-image: url(:/images/images/WHITE.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:rgb(255, 255, 255);
color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>marque</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
